File: AppGiaoHangAPI.Model/Model/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppGiaoHangAPI.Model.Model
{
    public class Account
    {
        public long AccountID { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public DateTime AccountCreateTime { get; set; }
        public string Roles { get; set; }
        public long EmployeeID { get; set; }

        public Employee Employee { get; set; }
    }
}

File: AppGiaoHangAndroid/app/src/main/java/com/pnb309/appgiaohang_android/Entity/Account.java
package com.pnb309.appgiaohang_android.Entity;

import android.os.Bundle;
import android.os.Parcelable;

import java.io.Serializable;
import java.util.Date;

public class Account implements Serializable {
    private long AccountId;
    private String UserName;
    private String Password;
    private Date AccountCreateTime;
    private String Roles;
    private Long EmployeeId;
    private Employee Employee;

    // Getters and Setters
    public long getAccountId() {
        return AccountId;
    }

    public void setAccountId(long accountId) {
        AccountId = accountId;
    }

    public String getUserName() {
        return UserName;
    }

    public void setUserName(String userName) {
        UserName = userName;
    }

    public String getPassword() {
        return Password;
    }

    public void setPassword(String password) {
        Password = password;
    }

    public Date getAccountCreateTime() {
        return AccountCreateTime;
    }

    public void setAccountCreateTime(Date accountCreateTime) {
        AccountCreateTime = accountCreateTime;
    }

    public String getRoles() {
        return Roles;
    }

    public void setRoles(String roles) {
        Roles = roles;
    }

    public Long getEmployeeId() {
        return EmployeeId;
    }

    public void setEmployeeId(Long employeeId) {
        EmployeeId = employeeId;
    }

    public Employee getEmployee() {
        return Employee;
    }

    public void setEmployee(Employee employee) {
        Employee = employee;
    }
}

